src/nwbinspector/_configuration.py

```diff
@@ -47,26 +46,28 @@ def copy_check(check):
     see https://github.com/NeurodataWithoutBorders/nwbinspector/pull/218 for explanation.
     """
     copied_check = _copy_function(function=check)
-    copied_check.__wrapped__ = _copy_function(function=check.__wrapped__)
+    copied_check.__wrapped__ = _copy_function(function=check.__wrapped__)  # type: ignore
+
     return copied_check
 
 
-def load_config(filepath_or_keyword: PathType) -> dict:
+def load_config(filepath_or_keyword: Union[str, Path]) -> dict:
     """
     Load a config dictionary either via keyword search of the internal configs, or an explicit filepath.
 
     Currently supported keywords are:
         - 'dandi'
             For all DANDI archive related practices, including validation and upload.
     """
-    file = INTERNAL_CONFIGS.get(filepath_or_keyword, filepath_or_keyword)
+    file = INTERNAL_CONFIGS.get(str(filepath_or_keyword), filepath_or_keyword)
     with open(file=file, mode="r") as stream:
         config = yaml.safe_load(stream=stream)
+
     return config
 
 
 def configure_checks(
-    checks: list = available_checks,
+    checks: Optional[list] = None,
```

Comment:
This is something I also adjusted while I was here

Not only did it make Mypy feel better but it has been a long-standing issue with the API docs (since they then render the full listing of named check functions which spams the signature)